src/main/java/com/sandvoxel/generitech/common/tileentities/power/TestPower.java

```diff
@@ -235,56 +70,14 @@ public NBTTagCompound writeToNBT(NBTTagCompound nbtTagCompound) {
 
 
     @Override
-    public IInventory getInternalInventory() {
-        return inventory;
-    }
-
-    @Override
-    public void onChangeInventory(IInventory inv, int slot, InventoryOperation operation, ItemStack removed, ItemStack added) {
-
-    }
-
-    @Override
-    public int[] getAccessibleSlotsBySide(EnumFacing side) {
-        return new int[0];
-    }
-
-    @Nullable
-    @Override
-    public ItemStack removeStackFromSlot(int index) {
-        return null;
-    }
-
-
-    @Override
     public List<String> getWailaBodyToolTip(ItemStack itemStack, List<String> currentTip, IWailaDataAccessor accessor, IWailaConfigHandler config) {
 
-        currentTip.add(LanguageHelper.LABEL.translateMessage("Power") + Long.toString(container.getStoredPower()));
+        currentTip.add(LanguageHelper.LABEL.translateMessage("Power ") + Long.toString(container.getStoredPower()));
 
         return currentTip;
     }
 
     @Override
-    public long givePower(long power, boolean simulated) {
-        System.out.println("packet sent :" + power);
-        this.markForUpdate();
-        this.markDirty();
-        container.givePower(power, simulated);
-        return power;
-    }
-
-    @Override
-    public long getStoredPower() {
-        return container.getStoredPower();
-    }
-
-    @Override
-    public long getCapacity() {
-        return container.getCapacity();
-    }
-
-
-    @Override
     @SuppressWarnings("unchecked")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS134)